routes/login.js

```diff
@@ -0,0 +1,39 @@
+const layout = require('../layout');
+
+const get = (req, res) => {
+    res.send(layout(`login`, `
+    <h1>Log In</h1>
+    <form method="POST">
+    <label for="username">Your username</label>
+    <input id="name" name="username"></input>
+    <label for="password">Your password</label>
+    <input id="password" name="password" type="password"></input>
+    <button type="submit">Submit</button>
+    </form>`));
+}
+
+function post(req, res) {
+  const {username, password } = req.body;
+
+  auth
+    .verifyUser(username, password)
+    .then((verification) => {
+        if(verification === false) { 
+            throw new Error();
+
+        } else {
+            return auth.createSession(verification);
+        }
+    })
+     .then((sid) => {
+         res.cookie("sis, sid, auth.COOKIE_OPTION");
```

Comment:
is 'sis' a typo?

Reply:
The ``.cookie`` method is expecting three arguments here:

1. The cookie name (aka the "key"): this could be anything, but we'll need to read it later, and it makes sense for it to be called ``"sid"``
2. The cookie data (aka the "value"). This needs to be whatever variable name holds the random sid string (confusingly, also ``sid``, but without quotes this time).
3. Cookie options. These will eventually be held by ``COOKIE_OPTION`` in ``auth.js``.

Long story short, you just need to move the second double quote :)